Cisco umbrella modeling rules (#30468)
demisto/content: Packs/Cisco-umbrella-cloud-security/README.md

<~XSIAM>
# Cisco Umbrella cloud security

### Collect Access Logs from Cisco Umbrella cloud security

#### AWS - S3 bucket

**On AWS**

First, make sure you have the below prerequisites:
1. A login to Amazon AWS service
2. A bucket configured in Amazon S3 to be used for storing logs.

See this[doc](https://docs.umbrella.com/deployment-umbrella/docs/setting-up-an-amazon-s3-bucket#prerequisites) for further instructions on how to create a S3 bucket, and how to configure it for your needs.

**On Cisco Umbrella cloud security**

1. Navigate to **Admin** -> **Log Management** and select **Use your company-managed Amazon S3 bucket**.
2. In the **Bucket Name** put the bucket name you created in Amazon S3 and click **Verify**.

> 3. After Cisco Umbrella verifies your bucket, it saves file called **README_FROM_UMBRELLA.txt** to it. 

Reviewer: ```suggestion
3. After Cisco Umbrella verifies your bucket, it saves a file called **README_FROM_UMBRELLA.txt** in the bucket. 
```

   Open the file and copy and paste the token listed in it into **Token Number** and click **Save**.

For more information, refer to the official Cisco Umbrella [documentation](https://docs.umbrella.com/deployment-umbrella/docs/setting-up-an-amazon-s3-bucket#enable-logging).

At the end of this process, you should have a folder created for each type of log in your bucket:
- auditlogs
- dnslogs
- proxylogs

Note:
Make sure that the Log schema version is configure to **v8**.
More information can be found [here](https://docs.umbrella.com/deployment-umbrella/docs/log-formats-and-versioning)

> **On XSIAM:**

Reviewer: ```suggestion
**On Cortex XSIAM:**
```

> 1. Navigate to **Settings** -> **Data Sources** -> **Add Data Source**.

Reviewer: ```suggestion
1. Navigate to **Settings** > **Data Sources** > **Add Data Source**.
```
